README.md

```diff
@@ -84,14 +84,73 @@ module.exports = {
 It is also possible to disable specific rules (or all rules) in a template itself:
 
 ```hbs
-{{! disable all rules for this template }}
-<!-- template-lint disable=true -->
+<!-- disable all rules -->
+{{! template-lint-disable }}
 
-{{! disable specific rules for this template }}
-<!-- template-lint bare-strings=false -->
+<!-- disable only bare-strings -->
+{{! template-lint-disable bare-strings }}
+
+<!-- disable only bare-strings and triple-curlies -->
+{{! template-lint-disable bare-strings triple-curlies }}
+
+<!-- enable all rules -->
```

Comment:
Just to confirm: this enables the rules as configured in the prior config stack, right?

```hbs
<div>
  {{! template-lint-disable bare-string }}
  <div>
    {{! template-lint-disable block-indentation }}
    <div>
      {{! template-lint-enable }} <b>what is the rule set now?</b>
    </div>
  </div>
</div>
```